engine/class_modules/monk/sc_monk.cpp

```diff
@@ -4065,6 +4140,61 @@ struct diffuse_magic_t : public monk_spell_t
 // Invoke Xuen, the White Tiger
 // ==========================================================================
 
+// Courage of the White Tiger ability ===================================================
+struct courage_of_the_white_tiger_t : public monk_melee_attack_t
+{
+  struct courage_of_the_white_tiger_heal_t : public monk_heal_t
+  {
+    courage_of_the_white_tiger_heal_t( monk_t *p )
+      : monk_heal_t( p, "courage_of_the_white_tiger_heal",
+                     p->talent.conduit_of_the_celestials.courage_of_the_white_tiger_heal )
+    {
+      background = dual = true;
+      target            = p;
+      base_dd_min = base_dd_max = 1.0;
+    }
+  };
+
+  courage_of_the_white_tiger_heal_t *heal;
+
+  courage_of_the_white_tiger_t( monk_t *p )
+    : monk_melee_attack_t( p, "courage_of_the_white_tiger_dmg",
+                           p->talent.conduit_of_the_celestials.courage_of_the_white_tiger_damage ),
+      heal( new courage_of_the_white_tiger_heal_t( p ) )
+  {
+    background = true;
+  }
+
+  double composite_aoe_multiplier( const action_state_t *state ) const override
+  {
+    double cam = monk_melee_attack_t::composite_aoe_multiplier( state );
+
+    if ( p()->buff.unity_within->check() )
+      cam *= 1 + p()->talent.conduit_of_the_celestials.unity_within_dmg_mult->effectN( 1 ).percent();
+
+    return cam;
+  }
```

Comment:
Not dependent on state, so a different override would be more appropriate.

Can be autoparsed-ish. Refer to implementation of `scalding_brew` autoparsing in `keg_smash_t`.